haddock-api/src/Haddock/Backends/Hyperlinker.hs

```diff
@@ -18,7 +18,7 @@ import Data.Maybe
 import System.Directory
 import System.FilePath
 
-import GHC.Iface.Ext.Types  ( HieFile(..), HieASTs(..), HieAST(..), SourcedNodeInfo(..) )
+import GHC.Iface.Ext.Types
```

Comment:
Is the amount of stuff imported from Ext.Types too large to be explicitly listed?

Reply:
I couldn't manage to import HiePath. It's just me not knowing how to import a pattern synonym ^^

Reply:
If I'm not mistaken: 

> Pattern synonyms can be exported and imported by prefixing the conid with the keyword pattern:

(_from https://gitlab.haskell.org/ghc/ghc/-/wikis/pattern-synonyms_)

Could you try this:

```suggestion
import GHC.Iface.Ext.Types (pattern HiePath, HieFile(..), HieASTs(..), HieAST(..), SourcedNodeInfo(..)) 
```

?

Reply:
Ok now I know! Fixed